tests/kuttl/tests/deployments/02-assert-conductor-groups.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  name: ironic
+  namespace: openstack
+spec:
+  serviceUser: ironic
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  storageClass: local-storage
+  ironicAPI:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
+  ironicConductors:
+  - replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    storageRequest: 10G
+  - conductorGroup: auckland
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    storageRequest: 10G
+    storageClass: standard-csi-auckland
```

Comment:
Should this change?

Reply:
Similar to what I mentioned above, I think we generally do not check `storage*` fields